docs/core/install/linux-package-manager-opensuse15.md

```diff
@@ -36,6 +36,13 @@ sudo chown root:root /etc/zypp/repos.d/microsoft-prod.repo
 
 Update the products available for installation, then install the .NET Core SDK. In your terminal, run the following command.
```

Comment:
```suggestion
Update the products available for installation, then install the .NET Core SDK. In your terminal, run the following command:

```bash
sudo zypper install dotnet-sdk-3.1
```
```

Reply:
Suggest putting the Important box below the existing command, otherwise it gets separated from its intro. Same for the other two places.